tests/e2e/Exchange.test.ts

```diff
@@ -0,0 +1,291 @@
+import 'mocha'
+import * as chai from 'chai'
+import * as chaiAsPromised from 'chai-as-promised'
+import { TLNetwork } from '../../src/TLNetwork'
+import { config, keystore1, user1, keystore2 } from '../Fixtures'
+
+chai.use(chaiAsPromised)
+
+describe('e2e', () => {
```

Comment:
Maybe more readable with await? We want to go using await, async more anyways, do we? But not nessasary for merge, we can open a new task for migrating to use async await more in the code

Reply:
yes definitely wanted to do a big refactoring of the code and use `await` and `async` more. I will open an issue for this.